R/ping.R

```diff
@@ -12,6 +12,8 @@
 #' }
 ping <- function(url = get_ckanr_url(), as="logical", ...)
 {
-  res <- ckan_POST(url, 'site_read', ...)
-  switch(as, json = res, logical = jsd(res))
+  retval <- tryCatch({
```

Comment:
Oh, I accidentally do it right.

In R, `a <- 1` will return `1`. But you are right, I forgot to put `retval` at the end.